app/controllers/collection_versions_controller.rb

```diff
@@ -33,6 +33,9 @@ def edit
     # if we end up on the edit page for a first draft (non-deposited collection), redirect to first draft edit page
     redirect_to edit_first_draft_collection_path(collection_version.collection) if collection_version.first_draft?
 
+    # start with a blank version description for a new version (pre-fill for saved as draft)
+    collection_version.version_description = '' if collection_version.deposited?
```

Comment:
Can this go in the `create_new_version` method?

Reply:
I don't think so, because we need the form to render with blank value (and it is rendered in `edit`, which will fetch the previous verison's description).

Reply:
Tested moving to the `create_new_version` method and it fails the new test I wrote.

Reply:
The test is for a situation that would never arise if you move it into create_new_version